recipe/meta.yaml

```
@@ -24,11 +22,11 @@ requirements:
   host:
     - python
     - pip
-    - setuptools >=61.0.0,<69.3.0
+    - setuptools
```

```suggestion
    - setuptools >=61.0.0,<69.3.0
```
https://github.com/django/django/blob/5.2/pyproject.toml#L2

---

Oddly enough, qdding a pinning to `setuptools` breaks the Python 3.13 build. The pinned releases predates 72.10, the first to support 3.13. Leaving setuptools unpinned allows for Django 5.2 to build cleanly on Python 3.13.